Formula/abricate.rb

```diff
@@ -0,0 +1,57 @@
+class Abricate < Formula
+  desc "Find antimicrobial resistance and virulence genes in contigs"
+  homepage "https://github.com/tseemann/abricate"
+  url "https://github.com/tseemann/abricate/archive/v1.0.1.tar.gz"
+  sha256 "5edc6b45a0ff73dcb4f1489a64cb3385d065a6f29185406197379522226a5d20"
+  license "GPL-2.0-only"
+  head "https://github.com/tseemann/abricate.git", branch: "master"
+
+  livecheck do
+    url :stable
+  end
```

Comment:
We can remove the `livecheck` block entirely because this is what happens by default (`stable` is checked first).